thrall/app/lib/ElasticSearch.scala

```diff
@@ -103,6 +103,7 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
           updateLastModifiedScript,
         scriptType)
       .executeAndLog(s"updating usages on image $id")
+      .recover { case e: DocumentMissingException => {}}
```

Comment:
@theefer `.executeAndLog(s"updating usages on image $id")` does log a descriptive error when it fails already which is why I removed the previous log statement from here as I felt it was redundant.

@jamesgorrie going to release this as is. I'd prefer not to return `Future[Any]` though so maybe you could explain your solution in person when you're around?